### PR TITLE
Dynamic pop-up window

### PR DESCRIPTION
**Pull request by VuDej**
Hi, 
I made a dynamic pop-up window in my Portfolio.
I deleted all projects section HTML so now the projects section is on load. Same for the pop-up window. I used a loop to iterate through objects and took properties and values for objects to render cards and the first project which has a different layout. I changed descriptions from the original Figma so you can see now when the user clicks on the button 'see project' on a pop-up window will be shown image, description, title, and technologies for every project.

Update.
Hi, if I see your image of the right margin I see the title also is missing. If I check on my Pc, I see a different one. On branch pop-up where is a project for me right margin is ok and there are titles of project. If I am wrong please tell me what should I do.

### CODE REVIEW

File: js/pop-up.js [hunk 1]
```diff
@@ -1,213 +1,211 @@
...
+            </li>
+            <li class='pop-btns'>
+                <a href="https://getbootstrap.com/">${project.technologies[1]}</a>
+            </li>
+            <li class='pop-btns'>
+                <a href="https://rubyonrails.org/">${project.technologies[2]}</a>
+            </li>
+        </ul>
+            <div class="pop-flex1 p-0 m-0">
+                <img class="img-fluid pop-img" src="${project.img}" alt="">
```
Comment (enaburekhan) [inline js/pop-up.js:182]: - [ ] It will be nice to adjust the right margin so it can fit properly in mobile. see screen-shot:

image

**Review by enaburekhan — CHANGES_REQUESTED** [review]
Hi VuDej, 

Good job so far!

- [ ] Major highlights

- The codebase is clean, easy to read and understand.
- Passed linters checks

 
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Cathella — APPROVED** [review]
## 🟢 Status: Approved 🎉🎉

### Hi @VuDej ✋🏻, 

Great job working on the changes requested by the previous reviewer 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## The Good Points 👍🏻

✔️ Linters are set correctly in the project repository. 
✔️ When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
✔️ When the user clicks (or taps) the close (X) button, the popup disappears.
✔️ The popup is responsive, it fits mobile and desktop sizes properly.
✔️ The popup follows the design for both desktop and mobile version from the chosen template in Figma.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by VuDej** [comment]
merged pop-up branch

**Pull request merged**